cunumeric/module.py

```diff
@@ -1259,6 +1282,28 @@ def check_shape_dtype(
     return converted, ArrayInfo(ndim, shape, dtype)
 
 
+def _block(arr, cur_depth, depth):
+    if cur_depth < depth:
+        inputs = list(_block(each, cur_depth + 1, depth) for each in arr)
+    else:
+        inputs = list(convert_to_cunumeric_ndarray(inp) for inp in arr)
+
+    # this reshape of elements could be replaced
+    # w/ np.atleast_*d when they're implemented
+    # Computes the maximum number of dimensions for the concatenation
+    max_list = list(inp.ndim for inp in inputs)
+    max_list.append(1 + (depth - cur_depth))
+    max_ndim = _builtin_max(max_list)
```

Comment:
Can't this be inlined? i.e., `_builtin_max(1 + depth - cur_depth, *(inp.ndim for inp in inputs))` I don't see a value in creating a list here.

Reply:
Sure, changed the code as suggested.